packages/apps/job-launcher/server/src/modules/webhook/webhook.dto.ts

```diff
@@ -1,8 +1,24 @@
 import { ChainId } from '@human-protocol/sdk';
-import { ApiProperty } from '@nestjs/swagger';
-import { IsEnum, IsString, IsBoolean, IsOptional } from 'class-validator';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import {
+  IsEnum,
+  IsObject,
+  IsString,
+  IsBoolean,
+  IsOptional,
+} from 'class-validator';
 import { EventType, OracleType } from '../../common/enums/webhook';
 
+export class EventData {
```

Comment:
In this case event data should be something like:
```
{
  "reason": "Invalid manifest" (This reason is just an example)
}
```

Reply:
Added Invalid Manifest and Manifest Cannot Be Downloaded as 2 possible reasons